devito/parameters.py

```diff
@@ -91,8 +97,23 @@ def initialize(self):
     'DEVITO_DEBUG_COMPILER': 'debug_compiler',
 }
 
+configuration = Parameters("Devito-Configuration")
+"""The Devito configuration parameters."""
+
+configuration.add('travis_test', 0, [0, 1], lambda i: bool(i))
+# Set the Instruction Set Architecture (ISA)
+default_isa = 'cpp'
+ISAs = [None, 'cpp', 'avx', 'avx2', 'avx512', 'knc']
+configuration.add('isa', None, ISAs)
+# Set the CPU architecture (only codename)
+default_platform = 'intel64'
+PLATFORMs = [None, 'intel64', 'sandybridge', 'ivybridge', 'haswell',
+             'broadwell', 'skylake', 'knc', 'knl']
```

Comment:
Do we still support KNC?

Reply:
well, what can I say...there's an ``IntelMICCompiler``, and yask supports it, so I'd leave it there... for the time being. I guess..